src/fps/task/MultisigTask.sol

```diff
@@ -150,48 +137,39 @@ abstract contract MultisigTask is Test, Script, ITask {
         _buildStarted = false;
     }
 
-    /// @notice Initialize the task with task and network configuration
-    /// @param taskConfigFilePath Path to the task configuration file
-    /// @param networkConfigFilePath Path to the network configuration file
-    /// @param _addresses Address registry contract
-    function _init(string memory taskConfigFilePath, string memory networkConfigFilePath, Addresses _addresses)
-        internal
-    {
+    /// @notice abstract function to be implemented by the inheriting contract
+    /// specifies the safe address string to run the template from
+    function safeAddressString() public pure virtual returns (string memory);
+
+    /// @notice abstract function to be implemented by the inheriting contract
+    /// specifies the addresses that must have their storage written to
+    function taskStorageWrites() internal pure virtual returns (string[] memory);
+
+    /// @notice Runs the proposal with the given task and network configuration file paths. Sets the address registry, initializes the proposal and processes the proposal.
+    /// @param networkConfigFilePath The path to the network configuration file.
```

Comment:
I'm not sure we should be calling this the network config anymore and instead just config.